### ISSUE TITLE
No ACL set for integrations

### ISSUE THREAD

**Issue by wigman** [magento/magento2#9684]
It's currently not possible to set access to Integrations (API access) for Admin roles. They are only available when *Role Resources* is set to *all*
 
### Preconditions
1. install any version of Magento 2 (up till 2.1.6)

### Steps to reproduce
1. create admin role
2. set custom role resources
3. select all resources
4. add a user to the role
5. login as this user
6. search for 'System > Extentions > Integrations'
7. cry, because you can't find it

### Expected result
1. ideally you'd be able to grant users access without giving them full rights. For 3rd party integrators for example.

### Actual result
See reproduction steps above

### Solution
Add /vendor/magento/module-integration/etc/acl.xml with following content:
```
<?xml version="1.0"?>
<!--
/**
 * Copyright © 2013-2017 Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
-->
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Acl/etc/acl.xsd">
    <acl>
        <resources>
            <resource id="Magento_Backend::admin">
                <resource id="Magento_Backend::system">
                    <resource id="Magento_Integration::extensions" title="System Extensions" translate="title" sortOrder="30">
                        <resource id="Magento_Integration::integrations" title="System Integrations" translate="title" sortOrder="40" />
                    </resource>
                </resource>
            </resource>
        </resources>
    </acl>
</config>
```

**Comment by korostii** [comment]
Possible duplicate of #7146, #5803, #9337, #9312 and who knows how many more similar issue reports =|

**Comment by wigman** [comment]
@korostii sorry, those are not the same. there's no issue with the mechanisms, the rules are simply not set for this module and should be added.

**Comment by korostii** [comment]
Okay, thanks for the clarification.
FYI, if you know for sure that your piece of ML fixes it, feel free to also create a corresponding Pull Request. Those seem to get processed faster than issues recently.

**Comment by AlexWorking** [comment]
Hello, @wigman. An internal ticked MAGETWO-70222 was created based on Your issue. You'll be informed as the issue is fixed. Thanks for applying.

**Comment by magento-engcom-team** [comment]
@wigman, thank you for your report.
We've created internal ticket(s) MAGETWO-70222 to track progress on the issue.

**Comment by okorshenko** [comment]
Hi @wigman. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1323 by @magento-engcom-team in 2.3-develop branch
Related commit(s):
 - 809ed03393aeb55d51897c88561d9cdf1d0907a3
 - de9d15a247e165baef6f8138713077d80ac798ce
 - b322c098e8d39bfb2ef8a4287df7a5cfba4b2a84
 - 703d9e0a5d3367b6719b4445957438e066f497ed
 - ecfe9010ba5a0fc67e9c8a1fccbf202695d4ec3d
 - a90dbca28e49bf87b219bb9bcb091fb5a964e265
 - b4a7213a7efcd532d58bb600ad483709cc1e970d
 - 660c62a82a041c3f203c9e395e2bc62e7788940f
 - 8d48fd346467aa2267a97fc838d42c33a7399138
 - d0b525b16ba86b3bd59bb8e23adf3d9687de16df
 - 0dd84cd92c5c2d66471e3a4cef7768da33afdaf6
 - 6746da4c6b1b535043b79f7ccd5af91488008d34
 - 413f9394c659ca24dd942a23a908c8500088436e
 - 2eca1d0038d9ecf33c33f4bd9cff28ff4e20ceb3
 - f9976b9365b8d445f932274eb2160127918b888c
 - 4c77e4f5ebb0f7d82efb24a0f02ab7ef7773d8a5
 - 6c62727ca10389745b92ac683e5bb8deee7496f3
 - d8915032a9b00b93601f5dd8316272a5949b94f2
 - d10f55e4016b91dacb2991b44b5a65fbd5ccd7ab
 - e2ff943b8011aa6604e7280e9581f074ebd5ecb3
 - e27445da1e3039d898c53a3f8a9052094a892038
 - f0711edabca0ecc5daf384a838576822a6f4d4af
 - 79de0d6ad14e04ebcc645f6e336082cde6035423
 - a30789d9ec4bc536423b350509790cfc018c5691

The fix will be available with the upcoming 2.3.0 release.
